packages/did-ethr-resolver/src/implementations/ewSigner.ts

```diff
@@ -120,9 +120,11 @@ export class EwSigner extends Signer {
       // Signer could be returned by not actually have an address
       // Best to check now and "fail fast"
       await signer.getAddress();
-    } catch (err) {
-      err.message = `Error instantiating EwSigner from ethers Provider. Provider must have signer with address. ${err.message}`;
-      throw err;
+    } catch (error) {
+      if (error instanceof Error) {
+        error.message = `Error instantiating EwSigner from ethers Provider. Provider must have signer with address. ${error.message}`;
+        throw new Error(error.message);
```

Comment:
I think error should be thrown regardless of `error` type

Reply:
This was done, because the `TS` version upgrade threw error while build "Object is of type unknown". So it has to be given a type before using. And also `any` cannot be used now with latest TS version.

Reply:
But what if `error` is not of type Error? Should method return normally?

Reply:
IMO, a standard error should be instance of `Error` always and if someone wants to have custom error then it would be a custom message with a new instance of the `Error`. Also we are only making standard calls using libraries so it should be fine and would appreciate your suggestion to replace this implementation.